gradio/themes/app.py

```diff
@@ -73,7 +73,7 @@
 
                 def go(*args):
                     time.sleep(3)
-                    return "https://gradio.app/assets/img/header-image.jpg"
+                    return "https://i.ibb.co/6BgKdSj/groot.jpg"
```

Comment:
```suggestion
                    return "https://raw.githubusercontent.com/gradio-app/gradio/main/js/_website/src/assets/img/header-image.jpg"
```

Reply:
Let's make the same change then in `builder_app.py`! Otherwise this looks good to me! Thanks @deepkyu !

Reply:
Got it @freddyaboulton

Reply:
Fixed it @freddyaboulton !